client/src/components/Minimap.vue

```diff
@@ -15,14 +16,16 @@
 
 <script>
 import { mapState } from 'vuex';
+import * as PIXI from 'pixi.js';
+import { ease } from 'pixi-ease';
 
 export default {
     name: 'Minimap',
     data: () => {
         return {
             gridSize: 22,
             gridBorderSize: 2,
-            gridPadding: 6,
+            gridPadding: 2,
             mapHeight: 0,
             mapWidth: 0,
             areaTitle: 'Unknown'
```

Comment:
It's a good idea to put defaults in here even if the default is null. That way it's easy to see what variables exist for a given Vue object without having to read the code and see when something new gets added to the object. For example, I'm suggesting adding:
```
app: null,
mapContainer: null,
```